spi/common/verifiable-credentials-spi/src/main/java/org/eclipse/edc/iam/verifiablecredentials/spi/validation/TrustedIssuerRegistry.java

```diff
@@ -16,17 +16,38 @@
 
 import org.eclipse.edc.iam.verifiablecredentials.spi.model.Issuer;
 
-import java.util.Collection;
+import java.util.Set;
 
 /**
- * A list of trusted VC issuers
+ * Trusted issuer registry.
  */
 public interface TrustedIssuerRegistry {
 
-    void addIssuer(Issuer issuer);
+    String WILDCARD = "*";
+
+    /**
+     * Register a supported type for a trusted issuer.
+     *
+     * @param issuer the issuer
+     * @param type   the type
+     */
+    void register(Issuer issuer, String type);
```

Comment:
nit: In this class, I would call it `credentialType`, and improve the Javadoc for it. The term `type` is not explicit enough IMO.
nit: 
```suggestion
void register(Issuer issuer, String credentialType);
```